Mlem/Localizable.xcstrings

```diff
@@ -1,6 +1,12 @@
 {
   "sourceLanguage" : "en",
   "strings" : {
+    "" : {
+
+    },
+    " " : {
+
+    },
```

Comment:
Sometimes Xcode doesn't remove strings that were once part of the build but were later removed

```suggestion
```